src/ripple/app/misc/impl/ValidatorList.cpp

```diff
@@ -601,26 +611,34 @@ ValidatorList::calculateQuorum (std::size_t nTrustedKeys)
     // Oi,j > nj/2 + ni − qi + ti,j
     // ni - pi > (ni - pi + pj)/2 + ni − .8*ni + .2*ni
     // pi + pj < .2*ni
-    return std::ceil(nTrustedKeys * 0.8f);
+    std::size_t quorum = std::ceil(nTrustedKeys * 0.8f);
```

Comment:
This implicit `float` to `std::size_t` conversion can generate warnings with compilers and/or static analysis tools. Prefer:

`auto quorum = static_cast<std::size_t>(std::ceil(nTrustedKeys * 0.8f));`